packages/material/README.md

```diff
@@ -1,24 +1,29 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
+
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+
+## Material Renderers Package
+
+This is the JSON Forms Material Renderers Package. This package only contains renderers and must be combined with [JSON Forms React](https://www.npmjs.com/package/@jsonforms/react).
+
+You can combine [JSON Forms React](https://www.npmjs.com/package/@jsonforms/react) with other renderers too, for example with the [Vanilla Renderers](https://www.npmjs.com/package/@jsonforms/vanilla-renderers).
 
```

Comment:
There should be a minimal example of how to use this renderer set, i.e. which packages to install and a minimal code example. See the vue vanilla readme as a guide.